### ISSUE TITLE
The program is stuck in this line.

### ISSUE THREAD

**Issue by BenoitKAO** [Drnoodle/fastpose#6]
Thank you for your work.

However,
when I run:
python demo_2d.py ressources/inputs/videos/demo.mp4

The program is stuck in this line:
cls_confs = torch.nn.Softmax()(Variable(output[5:5+num_classes].transpose(0,1))).data

Any help is appreciated.

**Comment by palashag12** [comment]
I have the same issue did you ressolve it?
